tl-tl/node/whitepaper.md

```diff
@@ -1,64 +1,64 @@
+Gayunpaman, sa sistemang ito,ang mga mensahe ay sumailalim sa pagkawala, pagkasira, pagkawalang-kilos, at pag-uulit. Ganun din, ang pagkaka-ayos ng pag-papadala ay hindi kailangan na naaalinsunod sa pagkakaayos ng pagtanggap ng mga mensahe. Ang mga aktibidad ng mga noda ay maaring maging arbitraryo, sila ay pwedeng sumali at umalis sa network sa kahit na anong panahon; sila din ay pwedeng magbasura at pumeke ng impormasyon o sadyang titigil lang sa pagtatrabaho. Artipisyal o hindi aryipisyal na mga glitches ay pwede din mangyari.
 
-Integrity and Authenticity of information transmission are ensured with cryptography while senders must attach signatures to the hash value of the message sent. Here we define 〈𝑚〉<sub>𝜎𝑖</sub> is the message m’s digital signature from node i, while D(m) is the hash value of message m. Without special clarification, all signature referred to in this article are signatures to the message hash value. 
+Itegridad at pagiging tunay ng mga transmisyon ng impormasyon ay natitiyak sa pamamagitan ng kriptograpiya habang ang nagpapadala ay kailangang maglagay ng lagda sa halaga ng hash ng mensaheng pinadala. Dito ay binibigyan nating ng kahulugan na ang 〈𝑚〉<sub>𝜎𝑖</sub> mensaheng m’s digital na lagda galing sa i na noda, habang ang D(m) ay ang halaga ng hash ng mensaheng m. Kahit walang espesyal na paglilinaw, lahat ng lagda na tinutukoy sa artikulong ito ay mga lagda sa halaga hash ng mensahe.
 
-## The Algorithm 
+## Ang Algoritmp
 
-​Our algorithm ensures security as well as usability. With erroneous nodes in the consensus making no more than ⌊ (𝑛−1) / 3 ⌋ , the functionality and stability of the system is guaranteed. In it, 𝑛 = |𝑅| suggests the total number of nodes joined in the consensus making while R stands for the set of consensus nodes. Given 𝑓 = ⌊ (𝑛−1) / 3 ⌋ , f stands for the maximum number of erroneous nodes allowed in the system. In fact, the total ledger is maintained by bookkeeping nodes while ordinary nodes do not participate in the consensus making. This is to show the entire consensus making procedures.
+​Ang ating algoritmo tinitiyak ang segurida at at abilidad na itoy pwedeng gamitin. Dahil sa mga maling noda sa konsensus ng paggawa ng hindi hihigit sa ⌊ (𝑛−1) / 3 ⌋ , ang panksyonalidad at katatagan ng sistema ay masisigurado. Sa loob nito, 𝑛 = |𝑅| ay nagmumungkahi na ang lahat ng kabuuang bilang ng noda na kasali sa paggagawa ng konsesus habang ang R ay ang ibig sabihin ay pangkat ng mga noda ng konsensus. Bigay ang 𝑓 = ⌊ (𝑛−1) / 3 ⌋ , f ay ang tumatayong kahulugan para sa pinakamataas na bilang ng mga maling noda na pwede sa sistema. Sa katunayan, ang kabuuang talaan ay binabantayan at iniingatan ng mga noda na bookkeeping habang ang mga ordinaryong noda ay hindi sumasali sa paggagawa ng konsensus. Ito ay para ipakita ang pangkalahatang pamamaraan ng paggawa ng konsensus.
```

Comment:
Ang ating algoritmo ay nagtitiyak sa seguridad at pati na rin sa abilidad na ito'y pwedeng gamitin.

Reply:
*masisigurado to masisiguro